quarkchain/cluster/shard_state.py

```diff
@@ -556,10 +558,6 @@ def run_block(
                 )
                 raise e
 
-        # Put only half of block fee to coinbase address
-        check(evm_state.get_balance(evm_state.block_coinbase) >= evm_state.block_fee)
-        evm_state.delta_balance(evm_state.block_coinbase, -evm_state.block_fee // 2)
```

Comment:
moved this part of logic to 1) applying tx and 2) collecting x-shard fee, respectively